[dev] add VSCode IDE tutorial

techfreaque opened the pull request.

GuillaumeDSM commented on installation/developer-installation/octobot-developer-installation.md:
```diff
@@ -99,30 +98,12 @@ We recommend using [PyCharm](https://www.jetbrains.com/pycharm/) to navigate thr
-    * Manually go through each file listed in the commands bellow and install its requirements using the PyCharm requirements plugin
-    * In a new terminal, activate your the virtual environement create by PyCharm (venv directory) and then run the following commands
+* For each OctoBot's repository: install missing dependencies in requirements.txt and dev\_requirements.txt
+* To install these requirements on your PCcharm virtual environment, you can either:
```
```suggestion
* To install these requirements on your PyCharm virtual environment, you can either:
```

GuillaumeDSM commented on installation/developer-installation/octobot-developer-installation.md:
```diff
@@ -207,3 +206,121 @@ this script will install the OctoBot-Tentacles folder and run Octobot with all y
+![](../../.gitbook/assets/image.png)
+
+{% hint style="info" %}
+The VS Code launch config is locate in project-folder/.vscode/launch.json
```
should this be added to OctoBot ? I see you closed the PR

techfreaque replied:
We could add it, but it would be on the wrong place anyway. Needs to be in the root of the project folder. So I decided to not add it there.
But maybe add it to a new project folder repo where vscode and pycharm scripts and settings are

GuillaumeDSM reviewed (commented):
thanks for these improvements, I'm sure it will help a lot of people !

Herklos commented on installation/developer-installation/octobot-developer-installation.md:
```diff
@@ -99,30 +98,12 @@ We recommend using [PyCharm](https://www.jetbrains.com/pycharm/) to navigate thr
-* To install these requirements on your pycharm virtual environement, you can either:
-    * Manually go through each file listed in the commands bellow and install its requirements using the PyCharm requirements plugin
-    * In a new terminal, activate your the virtual environement create by PyCharm (venv directory) and then run the following commands
+* For each OctoBot's repository: install missing dependencies in requirements.txt and dev\_requirements.txt
```
```suggestion
* For each OctoBot's repository: install missing dependencies in requirements.txt and dev_requirements.txt
```

Herklos commented on installation/developer-installation/octobot-developer-installation.md:
```diff
@@ -207,3 +206,121 @@ this script will install the OctoBot-Tentacles folder and run Octobot with all y
+            "type": "python",
+            "request": "launch",
+            "env": {
+                "PYTHONPATH": "${workspaceFolder}/Async-Channel;${workspaceFolder}/OctoBot-Tentacles-Manager;${workspaceFolder}/OctoBot-Commons;${workspaceFolder}/OctoBot-Trading;${workspaceFolder}/OctoBot-Backtesting;${workspaceFolder}/OctoBot-evaluators;${workspaceFolder}/OctoBot-Services;${workspaceFolder}/trading-backend"
```
I don't think we should add trading-backend in the developer env. Users should not modify edit.

Herklos commented on installation/developer-installation/octobot-developer-installation.md:
```diff
@@ -207,3 +206,121 @@ this script will install the OctoBot-Tentacles folder and run Octobot with all y
+        "./OctoBot-evaluators",
+        "./OctoBot-Trading",
+        "./OctoBot-Tentacles-Manager",
+        "./trading-backend"
```
Same comment about trading-backend

Herklos reviewed (changes requested):
Really nice improvements! Thanks

techfreaque committed:
[dev] add VSCode IDE tutorial


tmp


tmp


tmp


tmp


Update installation/developer-installation/octobot-developer-installation.md

Co-authored-by: Guillaume De Saint Martin <guillaumemdsm@gmail.com>
Update installation/developer-installation/octobot-developer-installation.md

Co-authored-by: Herklos <herklos.dev@protonmail.com>
tmp

techfreaque commented:
i changed it to your suggestions and added some typo fixes
